docs/interpreting-experiments/experiment-progress-bar.md

```diff
@@ -1,12 +1,14 @@
 # Experiment Progress Bar
 
+The progress bar helps you understand how much data you have gathered and whether it makes sense to continue gathering more data.
+
 #### Minimum Detectable Effect
-The progress of an experiment is measured with respect to the minimum detectable effect (MDE). This is the minimum effect the experimenter is interested in being able to detect and can be set at the metric level.
+The progress of an experiment is measured with respect to the minimum detectable effect (MDE). This is the minimum effect the experimenter is interested in being able to detect and is set at the metric level. However, it is possible to override this and set it to a particular value at the experiment level.
 
 **Note:** A small MDE requires the experiment to run much longer to collect sufficient data.
 
 ### Progress Bar
-Each experiment has an associated progress bar at the top. The goal of the progress bar is to measure whether we have gathered sufficient data to detect a lift of the primary metric by the MDE with high probability.
+Each experiment has an associated progress bar at the top. The goal of the progress bar is to measure whether we have gathered sufficient data to detect a lift of the primary metric by a configurable MDE with high probability.
```

Comment:
I think the second sentence is a bit unclear. What about something like "The goal of the progress bar is to measure whether we have gathered enough data to make it highly likely that we can detect a meaningful lift in the primary metric, where _meaningful_ means a lift of at least the MDE value configured for that experiment and that metric."

Reply:
I like it!